src/wallet/rpcwallet.h

```diff
@@ -24,6 +25,17 @@ void RegisterWalletRPCCommands(CRPCTable &t);
  */
 std::shared_ptr<CWallet> GetWalletForJSONRPCRequest(const JSONRPCRequest& request);
 
+/**
+ * Figures out what external signer to use for a JSONRPCRequest.
+ *
+ * @param[in] request JSONRPCRequest that wishes to access a signer
+ * @param[pos] which argument contains the signer fingerprint. Optional, returns the first signer otherwise
+ * @param[pwallet] the wallet
```

Comment:
The comment format is incorrect here :-)

Reply:
Thanks, I'll look into how that's supposed to work...